src/components/ScrollGradient/ScrollGradient.js

```diff
@@ -181,8 +182,36 @@ class ScrollGradient extends Component {
       ...other
     } = this.props;
     const { position } = this.state;
-    const gradientRotation =
-      direction === ScrollGradient.ScrollDirection.X ? -90 : 0;
+
+    let rgbCode = null;
+    const colorPropWarning = `The \`color\` property "${color}" supplied to \`ScrollGradient\` is not recognized as a valid HEX, RGB, or RGBA color code.`;
+
+    // Check if hex value:
+    if (color.startsWith('#')) {
+      const colorObject = hexRgb(color);
+      rgbCode = `${colorObject.red}, ${colorObject.green}, ${colorObject.blue}`;
+    }
+    // If not hex, check if RGB or RGBA value:
+    else if (color.toLowerCase().startsWith('rgb')) {
+      // Get comma separated string:
+      const colorCode = color.substring(
+        color.indexOf('(') + 1,
+        color.indexOf(')')
+      );
+      // Generate array from comma separated string:
+      const colorCodeArray = colorCode.split(',');
+
+      // Make sure code is the right length:
+      if (colorCodeArray.length === 3 || colorCodeArray.length === 4) {
+        rgbCode = `${colorCodeArray[0]}, ${colorCodeArray[1]}, ${
+          colorCodeArray[2]
+        }`;
+      } else {
+        console.warn(colorPropWarning);
```

Comment:
I'm doing a couple extra steps here because I thought it might be useful to be helpful & tell people if the RGB/RGBA is not valid (i.e., too long or too short) 🤔 

So instead of just taking `colorCode` and using that as the `rgb` value, I processed it a little to verify that it has 3 or 4 parts (red, green, blue, and maybe alpha though alpha will be dropped).

Let me know if you disagree and think I should just run with `codeCode`, without checking its length.